includes/class-sensei-utils.php

```diff
@@ -2812,6 +2812,15 @@ public static function get_target_page_post_id_for_continue_url( $course_id, $us
 		}
 		return $course_id;
 	}
+
+	/**
+	 * Check if the current theme supports Full Site Editing (FSE).
+	 *
+	 * @return bool True if FSE is supported, false otherwise.
+	 */
+	public static function is_fse_theme() {
```

Comment:
We should probably add an `@since` here since it's a new function.

Reply:
Thanks! updated here https://github.com/Automattic/sensei/pull/7044/commits/e069698046ecd28388ab1253f8374a5e06d3a4d8